clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -3540,7 +3353,7 @@ public void testPrepareJoinAndRejoinAfterFailedRebalance() {
             MockTime time = new MockTime(1);
 
             // onJoinPrepare will be executed and onJoinComplete will not.
-            boolean res = coordinator.joinGroupIfNeeded(time.timer(2));
+            boolean res = coordinator.joinGroupIfNeeded(time.timer(100));
```

Comment:
because we add a `timer.update();` at end of onJoinPrepare, this test will be failed on my local machine. So i have to increase the timeout.